Mathlib/CategoryTheory/Limits/Preserves/Filtered.lean

```diff
@@ -22,23 +20,35 @@ open CategoryTheory.Functor
 
 namespace CategoryTheory.Limits
 
-universe v u₁ u₂ u₃
+universe w' w₂' w w₂ v u₁ u₂ u₃
 
 -- declare the `v`'s first; see `CategoryTheory.Category` for an explanation
 variable {C : Type u₁} [Category.{v} C]
 variable {D : Type u₂} [Category.{v} D]
 variable {E : Type u₃} [Category.{v} E]
 variable {J : Type v} [SmallCategory J] {K : J ⥤ C}
 
+section FilteredColimits
+
+section Preserves
+
+-- This should be used with explicit universe variables.
+/-- `PreservesFilteredColimitsOfSize.{w', w} F` means that `F` sends all colimit cocones over any
+filtered diagram `J ⥤ C` to colimit cocones, where `J : Type w` with `[Category.{w'} J]`. -/
+@[nolint checkUnivs, pp_with_univ]
+class PreservesFilteredColimitsOfSize (F : C ⥤ D) where
+  preserves_filtered_colimits :
+    ∀ (J : Type w) [Category.{w'} J] [IsFiltered J], PreservesColimitsOfShape J F
+
 /--
 A functor is said to preserve filtered colimits, if it preserves all colimits of shape `J`, where
-`J` is a filtered category.
+`J` is a filtered category which is small relative to the universe in which morphisms of the source
+live.
 -/
-class PreservesFilteredColimits (F : C ⥤ D) : Type max u₁ u₂ (v + 1) where
-  preserves_filtered_colimits :
-    ∀ (J : Type v) [SmallCategory J] [IsFiltered J], PreservesColimitsOfShape J F
+abbrev PreservesFilteredColimits (F : C ⥤ D) : Type max u₁ u₂ (v + 1) :=
+  PreservesFilteredColimitsOfSize.{v, v} F
 
-attribute [instance 100] PreservesFilteredColimits.preserves_filtered_colimits
+attribute [instance 100] PreservesFilteredColimitsOfSize.preserves_filtered_colimits
 
 instance (priority := 100) PreservesColimits.preservesFilteredColimits (F : C ⥤ D)
     [PreservesColimits F] : PreservesFilteredColimits F where
```

Comment:
Should not this be generalized to a `...OfSize` version? (Same remark for `ReflectsColimits` below.)